conf/application.conf

```diff
@@ -174,6 +174,11 @@ microservice {
       }
     }
 
+    nsi-qeury-account {
```

Comment:
I think this probably isn't needed, if Rob confirms that the hostname for Get Account is the same as for Create Account then we can use the existing `nsi` config instead.

Reply:
yes, waiting for the response from Rob